cmd/security-spire-config/seed_builtin_entries.sh

```diff
@@ -26,14 +26,16 @@ echo "SPIFFE_SERVER_SOCKET=${SPIFFE_SERVER_SOCKET}"
 echo "SPIFFE_EDGEX_SVID_BASE=${SPIFFE_EDGEX_SVID_BASE}"
 
 # add pre-authorized services into spire server entry
-for dockerservice in security-spiffe-token-provider notifications scheduler \
+for dockerservice in security-spiffe-token-provider support-notifications support-scheduler \
     device-bacnet device-camera device-grove device-modbus device-mqtt device-rest device-snmp \
     device-virtual device-rfid-llrp device-coap device-gpio \
     app-service-http-export app-service-mqtt-export app-service-sample app-rfid-llrp-inventory \
     app-service-external-mqtt-trigger; do
     # Temporary workaround because service name in dockerfile is not consistent with service key.
     # TAF scripts depend on legacy docker-compose service name. Fix in EdgeX 3.0.
-    service=`echo -n ${dockerservice} | sed -e 's/app-service-/app-/'` 
+    service=`echo -n ${dockerservice} | sed -e 's/app-service-/app-/'`
+    # support- services have the opposite problem.  service key is right, service name in docker isn't
```

Comment:
Same issue with the `core services`. Service name is `data` while the service key is `core-data'. Something we need to clean-up in EdgeX 3.0. I have update this issue to cover App, Core and Support service names.
https://github.com/edgexfoundry/edgex-compose/issues/231